### DISCUSSION TITLE
Hero 5 session bricked updating labs firmware v2.51.72

### DISCUSSION THREAD

**Discussion by daojiachifan** [gopro/labs#1016] [Q&A]
manually update with labs firmware and the H5s bricked. I tried to back to the official version, but failed. Does anyone know how to fix it?
Fall into a dead loop
![image](https://github.com/user-attachments/assets/4aa6a71c-f377-4c23-b53b-ed493443f757)
![image](https://github.com/user-attachments/assets/01e92081-dffd-45ff-a892-c0ab507c4431)

**Comment by dnewman-gpsw** [comment]
You will need to contact regular GoPro support, they may know a procedure to recover a H5 Session from a failed update.

**Reply by daojiachifan** [reply]
Thanks. I'll contact them on workday and hope there will be a reply.

**Reply by daojiachifan** [reply]
I contacted GoPro support but couldn't fix the camera by flashing the regular firmware as they suggested. They mentioned that with my previous information, I could receive a coupon to purchase a new camera. Thank you for your response.
